app/main.py

```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+from functools import wraps
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Any:
+    result = {}
+
+    @wraps(func)
+    def inner(*args: Any) -> Any:
```

Comment:
`wrapper` better name in this case 
```suggestion
    def wrapper(*args: Any) -> Any:
```